index.html

```diff
@@ -1,17 +1,27 @@
 <!DOCTYPE html>
```

Comment:
- [ ] I think you forgot to add **readme.md** file to your project. Next time make sure you have added your descriptive readme file before submitting 🍀😉

Reply:
Done 👍

Reply:
Note that the readme file is in this branch now not in the main branch yet, it will be there after merge